dotCMS/src/integration-test/java/com/dotcms/rest/api/v3/contenttype/FieldResourceTest.java

```diff
@@ -458,6 +459,137 @@ public void shouldMoveFields () throws DotSecurityException, DotDataException {
+
+    }
+
+    final List<Map<String, Object>> createTestFields (final String contentTypeId) {
+
+        final List<Map<String, Object>> fieldList = new ArrayList<>();
+        final Map<String, Object> rowField = CollectionsUtils.map("clazz" , "com.dotcms.contenttype.model.field.ImmutableRowField",
+                "contentTypeId", contentTypeId,
+                "dataType" , "SYSTEM",
+                "fieldContentTypeProperties", Collections.emptyList(),
+                "fieldType", "Row",
+                "fieldTypeLabel", "Row",
+                "fieldVariables", Collections.emptyList(),
+                "fixed", Boolean.FALSE,
+                "iDate", new Long(1584647620000l),
+                "id","a2d05a0b-5f4d-449d-9ca2-1117ab1ab086",
+                "indexed",  Boolean.FALSE,
+                "listed", Boolean.FALSE,
+                "modDate", new Long(1584647620000l),
+                "name", "fields-0",
+                "readOnly",Boolean.FALSE,
+                "required", Boolean.FALSE,
+                "searchable", Boolean.FALSE,
+                "sortOrder", new Integer(0),
+                "unique", Boolean.FALSE,
+                "variable", "fields0");
+
+        final Map<String, Object> columnField = CollectionsUtils.map("clazz" , "com.dotcms.contenttype.model.field.ImmutableColumnField",
+                "contentTypeId", contentTypeId,
+                "dataType" , "SYSTEM",
+                "fieldContentTypeProperties", Collections.emptyList(),
+                "fieldType", "Column",
+                "fieldTypeLabel", "Column",
+                "fieldVariables", Collections.emptyList(),
+                "fixed", Boolean.FALSE,
+                "iDate", new Long(1584647620000l),
+                "id","a2d05a0b-5f4d-449d-9ca2-1117ab1ab086",
+                "indexed",  Boolean.FALSE,
+                "listed", Boolean.FALSE,
+                "modDate", new Long(1584647620000l),
+                "name", "fields-1",
+                "readOnly",Boolean.FALSE,
+                "required", Boolean.FALSE,
+                "searchable", Boolean.FALSE,
+                "sortOrder", new Integer(0),
+                "unique", Boolean.FALSE,
+                "variable", "fields1");
+
+        final Map<String, Object> textField = CollectionsUtils.map("clazz" , "com.dotcms.contenttype.model.field.ImmutableTextField",
+                "name", "text",
+                "dataType" , "TEXT",
+                "regexCheck", "",
+                "defaultValue", "",
+                "hint", "",
+                "required", Boolean.FALSE,
+                "searchable", Boolean.FALSE,
+                "indexed", Boolean.FALSE,
+                "listed", Boolean.FALSE,
+                "unique", Boolean.FALSE);
+
+        final Map<String, Object> textDecimalField = CollectionsUtils.map("clazz" , "com.dotcms.contenttype.model.field.ImmutableTextField",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'CollectionsUtils.map' due to existing static import 'com.dotcms.util.CollectionsUtils.*'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5214528)